front/app/components/admin/ResourceList/SortableRow.test.tsx

```diff
@@ -103,15 +101,15 @@ describe('<SortableRow />: integration with <SortableList />', () => {
       </SortableList>
     );
 
-    const lastItem = screen.getByText('ITEM ID: _3');
-    const middleItem = screen.getByText('ITEM ID: _2');
+    // SortableRow can be both a dragged element as well as a dropzone
+    const dragElement = screen.getByText('ITEM ID: _3');
+    const dropZone = screen.getByText('ITEM ID: _2');
 
-    fireEvent.dragStart(lastItem);
-    fireEvent.dragEnter(middleItem);
-    fireEvent.dragOver(middleItem);
-    fireEvent.dragEnter(lastItem);
-    fireEvent.dragOver(lastItem);
-    fireEvent.drop(lastItem);
+    // Similar implementation as dragAndDrop (dragAndDrop.ts)
+    fireEvent.mouseDown(dragElement, { which: 1, button: 0 });
+    fireEvent.dragStart(dragElement);
+    fireEvent.drop(dragElement);
+    fireEvent.mouseUp(dropZone, { which: 1, button: 0 });
```

Comment:
Dragging but not dropping seemed an edge-case to me, so I didn't add it to the dragAndDrop util.